### DISCUSSION TITLE
bhyve Compatibility

### DISCUSSION THREAD

**Discussion by Chrisco93Main** [sebr/bhyve-home-assistant#56] [General]
Just wanted to give you all a heads up. I tested this HACS integration with the **Orbit 57925** and it is working without any issues.

**Comment by sebr** [comment]
Thanks so much @Chrisco93Main - I'll update the readme with this information.

**Comment by gauzilla** [comment]
Tested on a **Orbit 12 Station 57950** and so far so good.  Still a little too cold to test turning zones on and off.

**Comment by themichaelasher** [comment]
There doesn't appear to be any difference from the 21004, but confirming that the Orbit 21005 (HT25) works as well. 
[https://www.amazon.com/dp/B0758V2JQS](https://www.amazon.com/dp/B0758V2JQS)

**Comment by jazn888** [comment]
Hi, Any idea if this integration will work on the Orbit B-hyve 12 Station Indoor/Outdoor Controller model 96050. https://www.bunnings.com.au/orbit-b-hyve-12-station-indoor-outdoor-irrigation-controller_p0057139?store=2432&gclid=CjwKCAiAm7OMBhAQEiwArvGi3A8nYkhvBupdH9ovczZ1YAa4IbE-iNoSLFECHxP6ez3r5-5AnToDHBoCU_MQAvD_BwE&gclsrc=aw.ds

**Comment by sebr** [comment]
> Hi, Any idea if this integration will work on the Orbit B-hyve 12 Station Indoor/Outdoor Controller model 96050. https://www.bunnings.com.au/orbit-b-hyve-12-station-indoor-outdoor-irrigation-controller_p0057139?store=2432&gclid=CjwKCAiAm7OMBhAQEiwArvGi3A8nYkhvBupdH9ovczZ1YAa4IbE-iNoSLFECHxP6ez3r5-5AnToDHBoCU_MQAvD_BwE&gclsrc=aw.ds
> 
> 

Most likely yes, but YMMV. Bunnings is usually pretty good with returns if it doesn't work ;)

**Comment by aleck011mi** [comment]
Just here to give appreciation for this integration and let everyone know that this is working with the B-hyve XD Smart Hose Timer (v2 hose timer). So far I have tried it with the single zone and I have 4 zone timers I will also be hooking up shortly and will confirm that they work also.
